python/cudf/setup.py

```diff
@@ -11,7 +12,14 @@
 
 install_requires = ["numba", "cython"]
 
-cython_files = ["cudf/_lib/**/*.pyx"]
+cython_files = ["cudf/**/*.pyx"]
+
+CUDA_HOME = os.environ.get("CUDA_HOME", False)
+if not CUDA_HOME:
+    CUDA_HOME = os.path.dirname(os.path.dirname(shutil.which("cuda-gdb")))
```

Comment:
It's possible that the user doesn't have `cuda-gdb` on their path, so `CUDA_HOME` could still not be found at this point. Might want to check it again and if not found, raise an exception telling the user to set `CUDA_HOME`.

Reply:
It's also possible the user has set the `CUDA_HOME` environment variable to a path which doesn't exist. @shwina is this needed long term or is this just a temporary workaround similar to cub / libcudacxx below?

Reply:
@raydouglass any opposition to us tackling this in a follow up pr?

Reply:
@kkraus14 Nope, it's just a nice-to-have to error out early on in my opinion since the user will get a compile error later that should make it obvious the CUDA includes are missing.